app/main/views/suppliers.py

```diff
@@ -263,6 +264,56 @@ def edit_supplier_organisation_size():
     return render_template('suppliers/edit_supplier_organisation_size.html', form=form)
 
 
+@main.route('/vat-number/edit', methods=['GET', 'POST'])
+@login_required
+def edit_supplier_vat_number():
+    form = VatNumberForm()
+    try:
+        supplier = data_api_client.get_supplier(current_user.supplier_id)['suppliers']
+    except APIError as e:
+        abort(e.status_code)
+
+    if supplier.get("vatNumber"):
+        return (
+            render_template("suppliers/already_completed.html", completed_data_description="vat number"),
```

Comment:
Do we want to capitalise 'VAT', like so?

`completed_data_description="VAT number"`

Reply:
Yep I think so, but check with @constancecerf

Reply:
Yeah, good spot.